VirtualH89/Src/H89Operator.h

```diff
@@ -0,0 +1,38 @@
+/// \file H89Operator.h
+///
+/// The VirtualH89-side implementation of the H89 Operator's interface, for
+/// standard functions such as RESET and diskette muont/unmount, but also
+/// providing virtual features like debugging.
+///
+/// \date Feb 9, 2016
+/// \author Douglas Miller
+///
+
+#ifndef H89OPERATOR_H_
+#define H89OPERATOR_H_
+
+#include <assert.h>
+#include <string>
+
+#include "config.h"
+#include "GenericDiskDrive.h"
```

Comment:
To simplify include graphs, can you forward declare these classes, and have the include in the .cpp file.